docs/sdkx-xamarin/tracking.mdx

```diff
@@ -0,0 +1,187 @@
+
+Possible datatypes to be passed into the config are:
+
+| Type         | value  | Comments                                                                                                                                                      |
+| ------------ | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| "singleline" | string | Single line string with character limit of 255                                                                                                                |
+| "multiline"  | string | Multi line string with character limit of 100,000                                                                                                             |
+| "number"     | string | String representation of number. For eg. "12345"                                                                                                              |
+| "dropdown"   | string | Drop-down options should exist for the given Custom Issue Field on dashboard. Value should be one of the values specified on the dashbaord for that dropdown. |
+| "date"       | long   | Epoch time in milliseconds. For eg. `1505927361535`                                                                                                           |
+| "checkbox"   | string | String representation of boolean. For eg. "true" or "false". This corresponds to the checkbox type custom issue field on dashboard.                           |
+
+<Admonition type="info" title="Note">
+
+**On Custom Issue Fields keys & compatibility**
+
+- Custom Issue Fields must be created in the Helpshift Dashboard (Settings → Custom Issue Fields), otherwise they will be ignored. Read more [here](https://support.helpshift.com/kb/article/custom-issue-fields/).
+
+</Admonition>
+
+## Breadcrumbs {#breadcrumbs}
+
+Breadcrumbs can be used to track events or end-user actions. It helps you to add debugging information regarding end-user actions, which will be passed along with every new conversation on Helpshift’s Agent Dashboard. To leave breadcrumbs, use `LeaveBreadCrumb` API —
+
+For example:
+
+```java
+Helpshift.LeaveBreadCrumb("Any breadcrumb string");
+```
+
+Breadcrumbs are collected within the set breadcrumb limit. The limit is set under the **SDK Configurations** section for App Settings in the Helpshift’s Agent Dashboard. Breadcrumbs are collected in a FIFO queue. If you want to clear the breadcrumbs, use the `ClearBreadCrumbs` API —
+
+For example:
+
+```java
+Helpshift.ClearBreadCrumbs();
+```
+
+<Admonition type="info" title="Note">
+
+- In the `LeaveBreadcrumb` API, passing of formatted strings is NOT supported. It is recommended to pass data only in a simple String format. For example:
+  - `"{\"key\":\"value\"}"` - ** JSON String, This is NOT supported **
+  - `"key - value"` - ** Simple string, This is supported **
+
+</Admonition>
+
+## Debug Logs {#debug-logs}
```

Comment:
we should also put in the log length limits we have.

Reply:
It's already mentioned. This line - "Debug logs are collected within the set debug logs limit. The limit is set under the **SDK Configurations** section for App Settings in the Helpshift’s Agent Dashboard. Debug logs are collected in a FIFO queue."